workspaces/linux-host/Makefile

```diff
@@ -69,6 +69,7 @@ test-client: test-dependencies
 veracruz-test: test-dependencies build-tests
 	RUST_LOG="integration_test=info,error" $(CARGO_TEST) --test integration_test -- --test-threads=1 --nocapture
 
+.PHONY: $(RUNTIME_ENCLAVE_BINARY_PATH)
```

Comment:
That is not consistent with the changes in IceCap. We should probably align on having a `.PHONY` target `<something>-runtime` rather that the runtime path as the dependency.

Reply:
Yes, I'll try that.